servicetest/cgroups/test_cgroups.py

```diff
@@ -54,40 +54,50 @@ def mounted_path_in_host():
     Container.READONLY: False
 }
 
-CONFIG_TEST_SMALL_THRESHOLD = [
-    {"setting": "memory.limit_in_bytes", "value": "1K"}
+CONFIG_TEST_MEMORY_SMALL_THRESHOLD = [
+    { "setting": "memory.limit_in_bytes", "value": "1K" }
```

Comment:
Spaces after the braces is not how the rest of the code looks, please align with the rest of the code or change everywhere.